Doc/c-api/module.rst

```diff
@@ -484,6 +484,12 @@ state:
 
    Add a string constant to *module*.
 
+.. c:function:: int _PyModule_AddType(PyObject *module, PyTypeObject *type)
+
+   Add an type object to *module*. The name of the type object is taken from
+   the last component of :c:member:`~PyTypeObject.tp_name` after dot.
+   Return ``-1`` on error, ``0`` on success.
```

Comment:
Mention also that PyType_Ready() is called on the type. Maybe mention that this function combines PyType_Ready(), _PyType_Name() and PyModule_AddObject().